src/LinuxTracingIntegrationTests/OrbitServiceIntegrationTest.cpp

```diff
@@ -44,9 +44,9 @@ int OrbitServiceMain() {
   LOG("OrbitService started");
   std::atomic<bool> exit_requested = false;
   // OrbitService's loop terminates when EOF is received, no need to manipulate exit_requested.
-  orbit_service::OrbitService{kOrbitServicePort, /*dev_mode=*/false}.Run(&exit_requested);
-  LOG("OrbitService finished");
-  return 0;
+  int rc = orbit_service::OrbitService{kOrbitServicePort, /*dev_mode=*/false}.Run(&exit_requested);
+  LOG("OrbitService finished with return code %d", rc);
+  return rc;
```

Comment:
Isn't "exit code" more common? Or is it because `Run` is not `main`?

Reply:
I have not really thought about it tbh. I changed it to exit_code